core/validation/initial_validation/schemas.py

```diff
@@ -13,9 +13,19 @@
     AuthorisationCheck,
     BasicCheck,
     ConflictingCheck,
+    SheetCheck,
 )
 
 TF_ROUND_4_INIT_VAL_SCHEMA = [
+    SheetCheck(
+        sheet="1 - Start Here",
+        error_message=(
+            "The data return template you are submitting is not valid. Please make sure you are submitting a valid "
+            "template for Towns Fund."
+            "If you have selected the wrong fund type, you can change the fund by returning to the 'Submit monitoring "
+            "and evaluation data dashboard' and changing the fund type to Towns Fund before continuing."
```

Comment:
Also not sure how picky our content/design people are, but possibly we want fancy quotes here? `“”` instead of `''`? If we aren't generally consistent though then disregard this.

Reply:
I copy and pasted from a Slack message but looks like we are using fancy quotes, amended.